stories/Button/Button.story.mdx

```diff
@@ -0,0 +1,94 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { Header } from '../shared';
+import Button from '../../components/Button';
+import { 
+  Default, 
+  FullWidth, 
+  Centered, 
+  Secondary, 
+  WithInfoIcon, 
+  WithSearchIcon, 
+  WithAccessibleIcon, 
+  WithErrorIcon, 
+  WithCreditCardIcon, 
+  WithLockedIcon, 
+  WithDoneIcon 
+} from './Button.stories.jsx';
+
+<Meta title="Button" component={Button}  />
+
+<Header title="Button">
+    Buttons express what action will occur when the user clicks
+    or touches it. Buttons are used to initialize an action, either in the
+    background or foreground of an experience.
+</Header>
+
+- [Importing and usage](#importing)
+- [API](#api)
+- [Examples](#examples)
+
+## <a id="importing"></a>Importing and usage
+
+```js
+import Button from '@catho/quantum/Button';
+```
+
+<Canvas>
+  <Story story={Default}/>
+</Canvas>
+
+## <a id="api"></a>API
+
+<ArgsTable of={Button} />
+
+## <a id="examples"></a>Examples
+
+### Full width
+
+<Canvas>
+  <Story story={FullWidth}/>
+</Canvas>
+
+### Centered
+
+<Canvas>
+  <Story story={Centered}/>
+</Canvas>
+
+### Secondary
+
+<Canvas>
+  <Story story={Secondary}/>
+</Canvas>
+
+### With Icons
+
+The full catalogue of icons can be found [here](?path=/docs/foundation-icons--page#catalog)
```

Comment:
Nice, but in `ArgsTable` link is incorrect.

Reply:
in this case the link should be without the #catalog?

Reply:
No, here is ok, but on the proptypes definition link should be adjusted